purchase_report_custom_hls/report/purchase_order_report_templates.xml

```diff
@@ -32,4 +35,16 @@
             </div>
         </xpath>
     </template>
+    <!-- Ideally, we should cut this line out into another module. -->
+    <template
+        id="report_purchaseorder_document"
+        inherit_id="purchase_stock.report_purchaseorder_document"
+    >
+        <xpath expr="//t[@t-else='']/t[@t-set='information_block']/strong" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+        <xpath expr="//div[@t-if='o.picking_type_id and o.picking_type_id.warehouse_id']" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
```

Comment:
I don't feel comfortable with this approach. If this and https://github.com/qrtl/hls-custom/pull/156/files#diff-2466a289dc9319e6af09eff092e75e482bf4775151dd4251c3f7db981849e854R7-R9 work, why not below?

```
        <xpath expr="//t[@t-else='']/t[@t-set='information_block']" position="attributes">
            <attribute name="invisible">1</attribute>
        </xpath>
```

Reply:
Sorry, I couldn't found.
I debug purchase_stock's models and add "t-att-style" in t-if or t-set tag. It doesn't works.

I want to merge this PR. After then, I will investigate why t-if or t-set aren't accepted any attribute.